tests/plugins/colordialog/manual/addhash.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: tc, feature, colordialog, 565, 4.8.0
+@bender-ui: collapsed
+@bender-ckeditor-plugins: table,tabletools,colordialog,toolbar,wysiwygarea,sourcearea,contextmenu
```

Comment:
It might be helpful when you wish to check what was included to attribute of tag. E.g. if there will be some invalid string or colour hard to distinguish from others, then checking source might be helpful.